BarrysPeanutsAkkaFlat/src/main/java/barryspeanuts/helper/MockHelper.java

```diff
@@ -0,0 +1,27 @@
+package barryspeanuts.helper;
+
+import barryspeanuts.model.Address;
+import barryspeanuts.model.CreditCard;
+import barryspeanuts.model.Customer;
+import java.util.UUID;
+
+public class MockHelper {
```

Comment:
Do you really need this class and entirely separate package for this trivial stuff? Can't you just instantiate things these where they are needed?

Reply:
Addressed, removed